addons/sys_intercom/vic3/data/Readme.md

```diff
@@ -0,0 +1 @@
+VIC3 FFCS models donated by RanTa to the ACRE Team. Any use outside ACRE project is explicitly forbidden without authorization.
```

Comment:
Same as GM stuff, we should modify license to support that and, like it is done in ACE3.

Separate component like ACE3's `apl`, with a different license that forbids any non-ACRE2 usage.